aten/src/ATen/native/Activation.cpp

```diff
@@ -216,15 +216,16 @@ Tensor& rrelu_with_noise_backward_out(
     const Tensor& noise,
     Scalar lower,
     Scalar upper,
-    bool training) {
+    bool training,
+    bool inplace) {
   auto lower_tensor = scalar_to_tensor(lower, grad_output.device());
   auto upper_tensor = scalar_to_tensor(upper, grad_output.device());
   if (training && (upper_tensor - lower_tensor).item().to<float>() > 1E-6) {
     grad_input = grad_output.mul(noise);
   } else {
     auto negative = (lower_tensor + upper_tensor) / 2;
     Scalar negative_slope = negative.item();
-    grad_input = at::leaky_relu_backward(grad_output, self, negative_slope);
+    grad_input = at::leaky_relu_backward(grad_output, self, negative_slope, inplace);
```

Comment:
wait, this shouldn't work you can't just reassign the parameter, you need to use the same memory.

Reply:
do we actually use this out function?

Reply:
removed